src/xml_parsing.cpp

```diff
@@ -319,6 +319,14 @@ void VerifyXML(const std::string& xml_text,
                                    "A Control node must have at least 1 child");
             }
         }
+        else if (StrEqual(name, "CheckerSequence"))
```

Comment:
this seems to be another change that must be here

Reply:
This is fine because it's a string check. `createTreeFromFile` will be called after we register new node types https://github.com/uml-robotics/fetchit/blob/master/main/include/main/Arena.h#L115

Reply:
this function here can also be done by writing a custom iterator to check if CheckerSequence has exactly 3 children

Reply:
See an example here: https://github.com/uml-robotics/fetchit/blob/master/main/include/main/bt/ExplainableBT.h#L129-L134

Reply:
Updated the check to iterate over children. Moved everything else to the PR here: https://github.com/SUCCESS-MURI/uml-byu-2021/pull/1